modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -160,9 +161,29 @@ SDA_remote_launcher <-function(settingPath,
       delete = FALSE,
       stderr = FALSE
     )
+    
+    # change the pft folder inside the setting
+    
+    settings$pfts %>%
+      purrr::map('outdir') %>%
+      walk(function(pft.dir) {
+        settings <<-
+          rapply(settings, function(x)
+            ifelse(
+              x == missing.input,
```

Comment:
Not clear where the value for `missing.input` is coming from at this point because it doesn't seem to get defined till later in this function.